### ISSUE TITLE
Print Symbol("true") as Symbol("true") instead of :true

### ISSUE THREAD

**Issue by Seelengrab** [JuliaLang/julia#31402]
This came up on slack when we were trying to create a `Symbol("true")` via `:true`.

```
julia> :true === true
true

julia> :true == Symbol("true")
false

julia> Symbol("true")
:true
```

@Keno proposed to change to printing to `Symbol("true")`, to make it clear that the symbol can't be created by using `:`. Further, `Symbol(true)` and the symbols of `false` should probably be handled the same, since the docstring for `Symbol` states that the string representation of the arguments is used to create the symbol.

```
help?> Symbol
search: Symbol

  Symbol(x...) -> Symbol

  Create a Symbol by concatenating the string representations of the arguments together.
```

**Comment by EricForgy** [comment]
Sounds good to me and would help explain:

```julia
julia> Symbol(true) == Symbol("true")
true
```

**Comment by StefanKarpinski** [comment]
Bug fix or minor change? I would probably lean towards minor change to be safe.

**Comment by Selich** [comment]
I will look into it if you don't mind.

**Comment by Naba7** [comment]
Can I be assigned this?

**Comment by fredrikekre** [comment]
There is already a PR: #32006

**Comment by JeffBezanson** [comment]
Fixed by #32006
